src/Components/WelcomePage/WelcomePage.js

```diff
@@ -15,17 +15,12 @@ export const WelcomePage = () => {
     .then(data=> {
       const addOne = [...displayedWorlds, data]
       dispatch(getDiscoveredWorlds(addOne))
-      // console.log(data.id)
       history.push(`/world/${data.id}`)
     }).catch((error)=>{
       console.log('This line replaced by error handling')
     })
```

Comment:
here's something I noticed! Let's get rid of this console log

Reply:
You got it, pushing up changes now - i both removed that console log and I added error handling on that line 20 to actually properly use your nifty error stuff